src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect, memo } from 'react';
+
+import { Loader } from '../Loader/Loader';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+import { getPeople } from '../../api';
+import { Person } from '../../types/Person';
+
+export const PeoplePage: React.FC = memo(() => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isLoadingFinish, setIsLoadingFinish] = useState(false);
+
+  const loadPeople = async () => {
+    try {
+      setIsLoading(true);
+
+      const loadedPeople = await getPeople();
+
+      setPeople(loadedPeople);
+    } catch (error) {
+      setIsError(true);
+    } finally {
+      setIsLoading(false);
+      setIsLoadingFinish(true);
+    }
+  };
+
+  useEffect(() => {
+    setIsError(false);
+    setIsLoadingFinish(false);
+    loadPeople();
+  }, []);
+
+  const isNoPeopleOnServer = isLoadingFinish && !isError && !people.length;
+  const isPeopleOnServer = isLoadingFinish && !isError && people.length;
```

Comment:
and don't forget about useMemo
```suggestion
  const isDataLoaded = !isLoading && !isError && people.length > 0;
```